packages/lib/link/connectors.js

```diff
@@ -102,12 +102,14 @@ class WebSocketBaseConnector extends events.EventEmitter {
 		if (!["handshake", "connected", "closing"].includes(this._state)) {
 			throw new libErrors.SessionLost("No session");
 		}
-		let message = { seq: this._seq, type, data };
+		let seq = this._seq;
```

Comment:
I'm curious - race conditions or just making code more readable and understandable? :)

Reply:
Readability - he increments this._seq, previously it was done in the return statement further down which was unclear.